text_file_functions.cpp

```diff
@@ -1,60 +1,85 @@
 #include "text_file_functions.h"
 
-QString AOApplication::read_theme()
+TextFileHandler& TextFileHandler::getInstance()
+{
+  static TextFileHandler instance;
+
+  // Create the QSettings class that points to the config.ini.
+  instance.configini = new QSettings(instance.get_base_path() + "config.ini", QSettings::IniFormat);
```

Comment:
Memory leak

Reply:
What part?
The `TextFileHandler` lives through the entirety of the program, so it will only need to get dumped out when the program terminates, unless I'm misunderstanding something.

Reply:
You fixed it, but basically every time you called `getInstance`, this code would replace `instance.configini` with a new instance of QSettings without deleting the old instance.

Reply:
Oh, I see!
I'll close this one, then.